torch/onnx/utils.py

```diff
@@ -1298,11 +1302,10 @@ def _find_typename(v):
                     "Got `%s`." % (type(v).__name__)
                 )
 
-        trace_module_map = __setup_trace_module_map()
         module_typenames = {_find_typename(v) for v in export_modules_as_functions}
         export_modules_as_functions = module_typenames
     else:
-        export_modules_as_functions = None
+        export_modules_as_functions = False
```

Comment:
I wonder if there is a reason `export_modules_as_functions` should be a bool if it's not a dict? I see it's later consumed by `_jit_pass_onnx_function_extraction`

Reply:
I think it was a mistake. Now it's renamed, and assigned as empty set.